### PR TITLE
Add captain/front and back

### PR DESCRIPTION
**Pull request by naremanhilles**
### Related:
related to #54 #68 

### Summary:
make add captain front end and backend

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -16,24 +19,36 @@ class App extends Component {
     return (
       <div className="App">
         <Router>
+          <ToastContainer
```
Comment (knanahassouna1) [inline client/src/App.jsx]: you don't need to react-tostify you can use antd.notifiaction

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import {Icon}from 'antd'
+import { Icon } from 'antd'
 import Customers from './Components/Layouts/Customers/index'
 import Sidebar from "./Components/CommonComponent/Sidebar";
 import Header from "./Components/CommonComponent/Header";
 import Navbar from "./Components/CommonComponent/Navbar/index";
```
Comment (knanahassouna1) [inline client/src/App.jsx]:
```suggestion
import Navbar from "./Components/CommonComponent/Navbar";
```

File: client/package.json [hunk 3]
```diff
@@ -18,12 +23,8 @@
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.0",
     "react-scripts": "3.0.1",
-    "sweetalert2": "^8.11.1",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-plugin-import": "^2.17.3",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.13.0"
+    "react-toastify": "^5.2.1",
```
[the remark this answers is not in the document]
Comment (naremanhilles) [reply client/package.json]: fexed it

File: client/src/App.jsx (hunk 2 above)
Comment (naremanhilles) [reply client/src/App.jsx]: fexed :+1:

File: client/src/App.jsx (hunk 1 above)
Comment (naremanhilles) [reply client/src/App.jsx]: remove tostify :+1:

**Pull request merged**